zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java

```diff
@@ -391,13 +387,13 @@ public Response insertParagraph(@PathParam("notebookId") String notebookId, Stri
    * @throws IOException
    */
   @GET
-  @Path("{notebookId}/paragraph/{paragraphId}")
+  @Path("{noteId}/paragraph/{paragraphId}")
   @ZeppelinApi
-  public Response getParagraph(@PathParam("notebookId") String notebookId,
-      @PathParam("paragraphId") String paragraphId) throws IOException {
-    LOG.info("get paragraph {} {}", notebookId, paragraphId);
+  public Response getParagraph(@PathParam("noteId") String noteId,
+                               @PathParam("paragraphId") String paragraphId) throws IOException {
```

Comment:
Thank you @jongyoul ! Let's make sure this and all other style issues are addressed before merging